portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm

```diff
@@ -104,16 +104,20 @@
                 </ul>
             </div>
             <div class="sites-section pinned-site-section">
-                <h3 class="sites-section-heading">${rloader.sitenav_title_pinned}</h3>
+                <h3 class="sites-section-heading" #if (! ${showSiteLabels} ) style="display: none;" #end>${rloader.sitenav_title_pinned}</h3>
```

Comment:
```suggestion
                <h3 class="sites-section-heading #if (! $showSiteLabels) d-none #end">${rloader.sitenav_title_pinned}</h3>
```
we can utilize the bootstrap class here, instead of inline style.